lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart

```diff
@@ -414,6 +447,7 @@ class HtmlViewEmbedder {
         }
       }
     }
+    _pictureRecordersCreatedDuringPreroll.clear();
```

Comment:
Do we need to call `PictureRecorder.delete()` somewhere? If so, do we?

Reply:
They are deleted when `endRecording` is called (in the above code)